ocean-client/src/programs/vault-maxi-program.ts

```diff
@@ -59,6 +63,28 @@ export class CheckedValues {
   }
 }
 
+class ReinvestTarget {
+  tokenName: string
+  percent: number | undefined
+  targetAddress: string | undefined
+  targetScript: Script | undefined
+  isCollateral: boolean
+
+  constructor(
+    tokenName: string,
+    percent: number | undefined,
+    isCollateral: boolean,
+    targetAddress: string | undefined = undefined,
+    targetScript: Script | undefined = undefined,
```

Comment:
if we would add targetVault too, maybe we should on parsing already decide what this reinvest targets wants to achieve aka introduce an enum